### PR TITLE
Feature/add nfg email for centralized storage

### PR DESCRIPTION
**Pull request by jonathanroehm**
Purpose of this feature:
Centralize the code for the email layouts that we wrote for nfg-branded emails so as to avoid bloating up our apps with email component partials, assets, etc. 

These files were originally created and are still present in their original form here: [sso_login_emails](https://github.com/network-for-good/sso_login_emails)

This branch does not aim to build a `ui.nfg.email` syntax, but instead to squarely place a lightweight solution so that evo can utilize inky partials to build out emails. 

Spec coverage is non-existent this round as these emails have been extensively tested, using these partials, from a previous initiative where their originating source code was used and is currently in use on their production environment (sso).

Example usage on Evo:

```ruby
= render partial: 'nfg_ui/email/header', locals: { heading: t('headings.hello', scope: i18n_scope, first_name: @resource.first_name) }
%row
  %columns
    %p= t 'body.intro', scope: i18n_scope, project_name: @project.name
    %p= t 'body.action', scope: i18n_scope
%row
  %columns
    %spacer{ size: spacer(:base) }
    = render partial: 'nfg_ui/email/button', locals: { url: dashboard_admin_project_url(@project, host: @resource.entity.email_host), text: t('buttons.continue', scope: i18n_scope), theme_color: theme_color(:success), size: :large }
    
    %spacer{ size: spacer(:base) }
    = render 'nfg_ui/email/email_signature', i18n_scope: i18n_scope
```

### CODE REVIEW

File: app/assets/stylesheets/nfg_ui/network_for_good/email/application.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
+@import "settings/settings";
+@import "foundation-emails";
+
+// @import "util/util";
```
Comment (jonathanroehm) [inline app/assets/stylesheets/nfg_ui/network_for_good/email/application.scss]: @kylejmorin any reason we should keep these commented out pieces at this point?

Comment (kylejmorin) [reply app/assets/stylesheets/nfg_ui/network_for_good/email/application.scss]: I can't remember why they were commented out besides the fact we didn't use or need them. We can always bring them back with version control. Cool to remove!

File: app/assets/stylesheets/nfg_ui/network_for_good/email/_global.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+/// @type Number
+$global-rounded: 500px !default;
+
+/// Global media query to switch from desktop to mobile styles.
+/// @type String
+$global-breakpoint: $global-width + $global-gutter !default;
+
+.wrapper {
+  width: 100%;
+}
```
Comment (kylejmorin) [inline app/assets/stylesheets/nfg_ui/network_for_good/email/_global.scss]: most of these styles can eventually be replaced with design tokens. woo!

File: config/locales/email.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        logo: "https://identity.networkforgood.org/images/nfg-logo.png"
+        alt:
+          caret: ">"
+          logo: "Network for Good"
+          social:
+            facebook: "Network for Good on Facebook"
+            twitter: "Network for Good on Twitter"
+            linkedin: "Network for Good on LinkedIn"
+            youtube: "Network for Good on YouTube"
+            google_plus: "Network for Good on Google+"
```
Comment (kylejmorin) [inline config/locales/email.yml]: I think we can remove google plus since we just removed it on evo campaign pages.

**Comment by jonathanroehm** [comment]
Note: this feature branch powers https://github.com/network-for-good/Givecorps-site/pull/1209

When https://github.com/network-for-good/Givecorps-site/pull/1209 is ready, this should be merged into master.

**Pull request merged**